devicetypes/smartthings/hue-bulb.src/hue-bulb.groovy

```diff
@@ -167,6 +168,10 @@ void setColorTemperature(value) {
     }
 }
 
+void installed() {
+	// setup device watch
+	sendEvent(name: "checkInterval", value: 60 * 6, displayed: false)
```

Comment:
You should add `data: [protocol: "lan"]` to this event. It tells device-watch that this device is lan connected, so it will not be marked offline whenever the TV extend is unplugged.